iOS_SDK/OneSignalSDK/Source/OSStateSynchronizer.m

```diff
@@ -237,27 +229,23 @@ - (void)sendPurchases:(NSArray *)purchases appId:(NSString *)appId {
     if (!_currentSubscriptionState.userId)
         return;
     
-    let pushStateSyncronizer = [self getPushStateSynchronizer];
-    let emailStateSyncronizer = [self getEmailStateSynchronizer];
-    
+    let stateSyncronizer = [self getStateSynchronizers];
+    // Begin constructing the request for the external id update
```

Comment:
This comment isn't correct, this is a sendPurchases update. Recommend removing instead of updating comment as the method name already describes the request about to be made.